subprojects/core/src/main/java/org/gradle/internal/composite/CompositeBuildSettingsLoader.java

```diff
@@ -65,4 +75,10 @@ public SettingsInternal findAndLoadSettings(GradleInternal gradle) {
 
         return settings;
     }
+
+    private void maybeInformAboutContinueOnFailureLimitation(StartParameter startParameter) {
+        if (startParameter.isContinueOnFailure()) {
+            LOGGER.warn("Using '--{}' with a composite build does not collect all failures.", ContinueOption.LONG_OPTION);
```

Comment:
It would be good to get in the habit of adding a link to our docs every place we write `logger.warn()`. This is not the best example, but if we don't have docs for each warning we should add them.

Reply:
If we are going to increasingly do this, we should invest in automation to detect broken links ahead of time and for released Gradle versions.